lib/src/main/java/io/ably/lib/transport/SecureSSLSocketFactory.java

```diff
@@ -0,0 +1,89 @@
+package io.ably.lib.transport;
+
+import javax.net.ssl.SSLSocket;
+import javax.net.ssl.SSLSocketFactory;
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.Socket;
+import java.net.UnknownHostException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * This is a decorator for the {@code SSLSocketFactory} which modifies the enabled TLS protocols
```

Comment:
Given `SSLSocketFactory` and `SSLSocket` are both imported, I think you can replace `@code` with `@link` to make these hyperlinks.

Reply:
Good suggestion, thanks :bow: https://github.com/ably/ably-java/pull/754/commits/28c3756185d101c4905d12c706ea720b0fc3b9fa